salt/migrations/2-3/etcd/pre-reboot.sls

```diff
@@ -0,0 +1,24 @@
+{% set roles = salt['grains.get']('roles', []) %}
+{% set has_etcd_role = ("etcd" in roles) %}
+
+{% if not has_etcd_role %}
+  # make sure there is nothing left in /var/lib/etcd
+
+cleanup-old-etcd-stuff:
+  cmd.run:
+    - name: rm -rf /var/lib/etcd/*
+
+uninstall-etcd:
```

Comment:
I think this is misleading, can you rename it to something like `disable-etcd`?

Reply:
Changing.